DESCRIPTION

```diff
@@ -23,6 +23,6 @@ License: GPL (>= 2)
 LazyLoad: yes
 LinkingTo: Rcpp
 Imports: Rcpp (>= 0.11.0), stats, utils, methods
-Suggests: RUnit, Matrix, pkgKitten, reticulate
+Suggests: RUnit, Matrix, pkgKitten, reticulate, highlight
```

Comment:
That is you deciding you need to throw this in. 

As I said yesterday, it does *not* throw a warning or note for me when checking, nor did CRAN comment on it.  As such it is not clearly needed.

It surely does not belong into a hotfix-PR about OpenMP.

Reply:
This is to address the build error happening on Ripley's machine. When we discussed this, the conclusion was to add it alongside of the OpenMP fix to save a PR.

Reply:
I doubt that. When we talked about the BDR failure we concluded it was a segfault neither one of us understood or was able to replicate.

In any event, the two commits were right and good, and I took it as opportunity to learn some more [magit](https://github.com/magit/magit) and cherry-picked them into a distinct PR which by now is in master.  So the vignette issue is taken care of.

Leaves 
- the macOS clusterfuck, and the valid point by @kevinushey
- the unexplicable fail on clang-on-fedora which I plan to simply ignore unless someone shows how to replicate